### PR TITLE
Added Count Good Triplets

### PR DESCRIPTION
**Pull request by manishh12**
## Related Issue

Fix: #437 

## Description

Added solution for count good triplets under leetcode solution section.

## Type of PR

- [ ] Bug fix
- [x] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): _______________

## Screenshots / Videos (if applicable)

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa-solutions/lc-solutions/1500-1599/1534-Count-good-triplets.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+---
+
+In this page, we will solve the Count Good Triplets problem using multiple approaches: brute-force and optimization. We will provide the implementation of the solution in Python, JavaScript, TypeScript, Java, and C++.
+
+## Problem Description
+
+Given an array of integers `arr`, and three integers `a`, `b`, and `c`. You need to find the number of good triplets.
+
+A triplet `(arr[i], arr[j], arr[k])` is good if the following conditions are true:
+
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/1500-1599/1534-Count-good-triplets.md:23]: try use like

```md
- $0 <= i < j < k < arr.length$
- $|arr[i] - arr[j]| <= a$
- $|arr[j] - arr[k]| <= b$
- $|arr[i] - arr[k]| <= c$
```

File: dsa-solutions/lc-solutions/1500-1599/1534-Count-good-triplets.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+description: "This is a solution to the Count Good Triplets problem on LeetCode."
+---
+
+In this page, we will solve the Count Good Triplets problem using multiple approaches: brute-force and optimization. We will provide the implementation of the solution in Python, JavaScript, TypeScript, Java, and C++.
+
+## Problem Description
+
+Given an array of integers `arr`, and three integers `a`, `b`, and `c`. You need to find the number of good triplets.
+
+A triplet `(arr[i], arr[j], arr[k])` is good if the following conditions are true:
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/1500-1599/1534-Count-good-triplets.md:22]: replace `(arr[i], arr[j], arr[k])` to `$(arr[i], arr[j], arr[k])$`

File: dsa-solutions/lc-solutions/1500-1599/1534-Count-good-triplets.md [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+
+```plaintext
+Input: arr = [1,1,2,2,3], a = 0, b = 0, c = 1
+Output: 0
+Explanation: No triplet satisfies all conditions.
+```
+
+### Constraints
+
+- `3 <= arr.length <= 100`
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/1500-1599/1534-Count-good-triplets.md:53]: also, change this

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
change code based on suggestions

**Comment by manishh12** [comment]
@Ajay-Dhangar Done kindly review it.

**Review by Ajay-Dhangar — APPROVED** [review]
done

**Pull request merged**